app/views/software_choices_filter.scala.html

```diff
@@ -83,6 +83,7 @@ <h1 class="govuk-heading-xl">@messages("softwareChoices.filter.title")</h1>
 
     @formWithCSRF(action = controllers.routes.SoftwareChoicesController.search, 'id -> "searchForm") {
         <div class="govuk-grid-row">
+            <h2 class="govuk-visually-hidden">@messages("softwareChoices.filter.filters")</h2>
```

Comment:
The "govuk-visually-hidden" class needs removing here - the heading should be displayed